source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -678,8 +693,14 @@ void JsonTranscoderFilter::buildResponseFromHttpBodyOutput(
   google::api::HttpBody http_body;
   for (auto& frame : frames) {
     if (frame.length_ > 0) {
+      http_body.Clear();
       Buffer::ZeroCopyInputStreamImpl stream(std::move(frame.data_));
-      http_body.ParseFromZeroCopyStream(&stream);
+      if (!HttpBodyUtils::parseMessageByFieldPath(&stream, method_->response_body_field_path,
```

Comment:
Doing there is only supporting HttpBody response.  What about normal JSON response?

You may have to implement it here

https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/blob/master/src/response_to_json_translator.cc#L57

Reply:
I can submit a version of this change to `grpc-httpjson-transcoding` library instead & use it both here for HttpBody and there for JSON.
Protobuf spec allows inclusion of the same message field multiple times with an expectation that parser merges them. In order to support this generically I may need to provide `ZeroCopyInputStream` implementation built on top of the chain of sub-streams. It can be somehow less efficient though hopefully still the same amount of copies.

Thoughts?

Reply:
Alternatively we could decide to not support this edge case and treat it as an error until somebody complains.

Reply:
We can make supporting `response_body` in HttpBody and JSON as two different features. This PR can be just for HttpBody,  and file another issue to track JSON feature.

Please log a proper warning or error code for not supported case.